src/carts/MagentoCartClient.js

```diff
@@ -36,10 +36,15 @@ class MagentoCartClient extends MagentoClientBase {
 
     /**
      * Gets a CCIF cart by id.
+     * This method accepts optional HTTP headers and status code which can be used when returning the cart after
+     * a POST and DELETE operation.
+     *
+     * @param headers optional HTTP response headers
+     * @param statusCode optional HTTP status code
      *
      * @return {Promise}   Promise with CCIF cart Object.
      */
-    get() {
+    get(headers, statusCode = 200) {
```

Comment:
I have to admit I'm not a big fan of this change ... it's more a "hack" because of that GET after POST that we do. I'd prefer to explicitly override the response in `postCartEntry` where you need this. This is exactly how you did it to add the `Location` header.